contrib/devtools/symbol-check.py

```diff
@@ -141,14 +142,17 @@
 }
 
 PE_ALLOWED_LIBRARIES = {
+# libbitcoinconsensus-0.dll, bitcoind.exe and bitcoin-qt.exe
 'ADVAPI32.dll', # security & registry
-'IPHLPAPI.DLL', # IP helper API
 'KERNEL32.dll', # win32 base APIs
 'msvcrt.dll', # C standard library for MSVC
+'libssp-0.dll', # mingw-w64 stack smashing protection
```

Comment:
NACK. This is accomodating a bug (in the DLL), and doing so in such a way, that it could leak into the actual binaries.

Reply:
> This is accomodating a bug...

Which one?

Reply:
Non-static libssp.